template.rb

```diff
@@ -276,7 +276,7 @@ def add_sitemap
   git commit: %Q{ -m 'Initial commit' }
 
   say
-  say "Jumpstart app successfully created! 🙌", :blue
+  say "Jumpstart app successfully created!", :blue
```

Comment:
Don't like my emoji? 😜

Reply:
Haha I love it but I was getting an Ascii error!

Reply:
Weird. Webpacker has one in it, so you must have the issue there then too.